Scripts/Core/EditorVR.Interfaces.cs

```diff
@@ -282,6 +293,27 @@ internal void ConnectInterfaces(object obj, Transform rayOrigin = null)
 				if (requestStencilRef != null)
 					requestStencilRef.requestStencilRef = RequestStencilRef;
 
+				var linkedTool = obj as ILinkedTool;
+				if (linkedTool != null)
+				{
+					List<ILinkedTool> otherTools;
+					var type = obj.GetType();
+					if (m_LinkedTools.TryGetValue(type, out otherTools))
+					{
+						foreach (var otherTool in otherTools)
+						{
+							otherTool.otherTools.Add(linkedTool);
```

Comment:
We should simply maintain a list of all tools per linked tool instead of trying to exclude the single tool and maintain all the lists separately.